src/pages/blog/2022-04-25-ako-na-monero-node-na-raspberry-pi.md

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
Prosim, pouzi `prettier` na sformatovanie suboru. Malo by to ist cez prikaz `npm run format`. Vdaka

Reply:
prikaz som zbehla